draft-ietf-masque-h3-datagram.md

```diff
@@ -343,8 +343,10 @@ identified fields or a capsule payload that terminates before the end of the
 identified fields MUST be treated as a malformed or incomplete message. In
 particular, redundant length encodings MUST be verified to be self-consistent.
 
-When a stream carrying capsules terminates cleanly, if the last capsule on the
-stream was truncated, this MUST be treated as a malformed or incomplete message.
+If a stream carrying capsules terminates cleanly (for example, in HTTP/3 this is
```

Comment:
could we say which side? E.g. 

if the receive-side of a stream terminates cleanly

Reply:
Good point, done